vnext/Microsoft.ReactNative/Views/ReactRootControl.cpp

```diff
@@ -269,39 +269,12 @@ void ReactRootControl::ShowInstanceLoaded(Mso::React::IReactInstance &reactInsta
   }
 }
 
-void ReactRootControl::ShowInstanceError(Mso::React::IReactInstance &reactInstance) noexcept {
-  if (!m_isDevModeEnabled)
-    return;
-
+void ReactRootControl::ShowInstanceError() noexcept {
```

Comment:
>ShowInstanceError [](start = 23, length = 17)

Should we remove it completely?

Reply:
It removes the loading state UI, or UI from previously running instances, which are both still probably good things to do.